### ISSUE TITLE
NullFieldMarshallingTest.testAbstractNullFieldUnmarshalledCorrectlyBinary is failing

### ISSUE THREAD

**Issue by alamar** [OpenHFT/Chronicle-Wire#346]
Before #341 it didn't fail, on 5c81549982dd23b21e4c44f815d043cdd86f0b6b it will always pass.

**Comment by alamar** [comment]
See for example https://teamcity.chronicle.software/buildConfiguration/OpenHFT_BuildAll_Build/629733

**Comment by alamar** [comment]
```
         public Object readUsing(Class clazz, Object o, @NotNull ValueIn in, BracketType bracketType) {
-            if (o == null)
-                throw new IllegalArgumentException("Unable to deserialize " + clazz);
             Wires.readMarshallable(clazz, o, in.wireIn(), true);
```
If these lines are removed, `NullFieldMarshallingTest` is fixed but now `Issue341Test` will start to fail.

**Comment by alamar** [comment]
See also #347

**Comment by hft-team-city** [comment]
Released in [Chronicle-Wire-2.22ea11](https://github.com/OpenHFT/Chronicle-Wire/releases/chronicle-wire-2.22ea11), [BOM-2.22ea69](https://github.com/OpenHFT/OpenHFT/releases/chronicle-bom-2.22ea69)
